README-cdk.md

```diff
@@ -224,17 +224,32 @@ oc create clusterrolebinding federation-admin \
           --serviceaccount="federation-system:default"
 ~~~
 
-Deploy the [cluster registry](https://github.com/kubernetes/cluster-registry) and the namespace where clusters are registered
-(`kube-multicluster-public`):
+Clone the Federation V2 repo and checkout the tag 0.0.4:
 
 ~~~sh
-oc create -f cluster-registry.yaml
+git clone https://github.com/kubernetes-sigs/federation-v2/ /var/tmp/federation-v2/
```

Comment:
Ditto about `/var/tmp`... it might be the right thing to do, as an easy way to have full path references elsewhere in the doc. But I'm just wondering if it's strictly necessary, and if not I'd vote to keep things relative.